modules/budget/variables.tf

```diff
@@ -53,6 +53,24 @@ variable "services" {
   default     = null
 }
 
+variable "calendar_period" {
+  description = "Specifies the calendar period for the budget. Possible values are MONTH, QUARTER, YEAR, CUSTOM, CALENDAR_PERIOD_UNSPECIFIED"
```

Comment:
Let's call out that `custom_period_start_date` and `custom_period_end_date` must be set if `CUSTOM`